jscatter/jscatter.py

```diff
@@ -2497,7 +2572,64 @@ def axes(
             labels = self._axes_labels,
         )
 
-        return self._mouse_mode
+    def legend(
+        self,
+        legend: Optional[Union[bool, Undefined]] = UNDEF,
+        position: Optional[Union[Position, Undefined]] = UNDEF,
```

Comment:
these signatures could be simplified as `Union[SomeType, Undefined, None]`. See https://fastapi.tiangolo.com/python-types/#using-union-or-optional. Fairly subjective point, but something I've adopted in projects recently.

EDIT: Ah... I see the style in the rest of this module uses `Optional` maybe something to think about for a follow-up PR but not necessary.

Reply:
Ha! That's interesting. I think I'll stick to `Optional` for now but I guess `Union[bool, Undefined]` would be sufficient as well. (`None` is not valid in this case. I remember we talked about this but in order to distinguish (elsewhere) between unsetting and ignoring a value I introduced `Undefined`. This way I can check if the user provided no value versus `None`.)